hexa/ui/tests/test_datagrid.py

```diff
@@ -24,10 +24,8 @@ class BikeDatagrid(Datagrid):
             )
             category = TextColumn(text="category")
             characteristics = TextColumn(text="suspension", secondary_text="group_set")
-            created_at = DateColumn(
-                date="created_at", date_format="%Y-%m-%d %H:%M:%S %Z"
-            )
-            launch_in = DateColumn(date="launch_in", date_format="timesince")
```

Comment:
The previous test was testing two different usages: one with a date format string, the other with `timesince`. Don't remove `timesince` and its test.

Existing tests should be changed very carefully, as you might get the wrong impression that "you are not breaking existing features because all tests pass". By changing the tests, you are taking a risk : breaking an existing feature without noticing it.

(I'm not saying that you can't change an existing test - just that you need to think about it carefully)

Reply:
However, in this case, I think this test should not be changed at all.